win32.js

```diff
@@ -44,8 +44,14 @@ module.exports = {
       if (opts.appCopyright) {
         rcOpts['version-string'].LegalCopyright = opts.appCopyright
       }
+      
+      // add "asInvoker" to the built manifest by default
+      rcOpts['requested-executation-level'] = 'asInvoker'
+      if (opts.win32exeLevel) {
```

Comment:
I've talked to @MarshallOfSound about this, we've come to the conclusion that it should go into `win32metadata`, despite it being not quite metadata. I'd like the property to be called `requestedExecutionLevel` since that's the name of the manifest property (according to that blog post. _(The other properties are the case-sensitive property names set by rcedit.)_